cli/src/constants/providers/index.ts

```diff
@@ -2,3 +2,4 @@
 export * from "./labels.js"
 export * from "./settings.js"
 export * from "./models.js"
+export * from "./zai.js"
```

Comment:
Let's exclude the changes of the Zai wizard improvements from this PR to isolate the changes.